src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -990,9 +990,9 @@ private function registerRequestConfiguration(array $config, ContainerBuilder $c
         }
     }
 
-    private function registerAssetsConfiguration(array $config, ContainerBuilder $container, XmlFileLoader $loader)
+    private function registerAssetsConfiguration(array $config, ContainerBuilder $container, PhpFileLoader $phpLoader)
```

Comment:
please don't change the variable name here. and you need to call the function with the new `$phpLoader` variable on top

Reply:
Thank for response. I have updated